### PR TITLE
Add Mozilla Code of Conduct

### LINKED ISSUE #298: CODE_OF_CONDUCT.md file missing
As of January 1 2019, Mozilla requires that all GitHub projects include this [CODE_OF_CONDUCT.md](https://github.com/mozilla/repo-templates/blob/master/templates/CODE_OF_CONDUCT.md) file in the project root. The file has two parts:

1. Required Text - All text under the headings *Community Participation Guidelines and How to Report*, are required, and should not be altered.
2. Optional Text - The Project Specific Etiquette heading provides a space to speak more specifically about ways people can work effectively and inclusively together. Some examples of those can be found on the [Firefox Debugger](https://github.com/devtools-html/debugger.html/blob/master/CODE_OF_CONDUCT.md) project, and [Common Voice](https://github.com/mozilla/voice-web/blob/master/CODE_OF_CONDUCT.md). (The optional part is commented out in the [raw template file](https://raw.githubusercontent.com/mozilla/repo-templates/blob/master/templates/CODE_OF_CONDUCT.md), and will not be visible until you modify and uncomment that part.)

If you have any questions about this file, or Code of Conduct policies
and procedures, please see
[Mozilla-GitHub-Standards](https://wiki.mozilla.org/GitHub/Repository_Requirements)
or email Mozilla-GitHub-Standards+CoC@mozilla.com.

_(Message COC001)_

### PR DESCRIPTION
**Pull request by Mozilla-GitHub-Standards**
Fixes #298

_(Message COC002)_

### CODE REVIEW

**Comment by hmitsch** [comment]
Hi @hidde, I hope it's okay that I set you as reviewer. Can your please merge this PR?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+This repository is governed by Mozilla's code of conduct and etiquette guidelines. 
+For more details, please read the
+[Mozilla Community Participation Guidelines](https://www.mozilla.org/about/governance/policies/participation/). 
+
+## How to Report
+For more information on how to report violations of the Community Participation Guidelines, please read our '[How to Report](https://www.mozilla.org/about/governance/policies/participation/reporting/)' page.
+
+<!--
+## Project Specific Etiquette
```
Comment (hidde) [inline CODE_OF_CONDUCT.md]: Should we remove this bit as we have no NLX specific etiquette?

Comment (hmitsch) [reply CODE_OF_CONDUCT.md]: Yes, we can do that.

**Comment by hidde** [comment]
Closing in favour of https://github.com/mozilla-iam/auth0-custom-lock/pull/300, which only has signed commits (which we need in order to merge)

**Pull request closed without merging**